Applications/Games/Wildlife Park 2/Steam/script.js

```diff
@@ -0,0 +1,19 @@
+include(["Engines", "Wine", "QuickScript", "SteamScript"]);
+include(["Engines", "Wine", "Verbs", "quartz"]);
+
+new SteamScript()
+    .name("Wildlife Park 2")
+    .editor("B-Alive")
+    .applicationHomepage("www.wildlifepark2.com")
+    .author("Zemogiter")
+    .appId(264710)
+    .executable("Steam.exe", ["-silent", "-applaunch", 304350, "-no-ces-sandbox"])
```

Comment:
probably: "-no-cef-sandbox"

However: that's default so this line is not needed.

Reply:
The "-no-ces-sandbox" argument is used in my Subnautica script and codacy didn't threw anything about it.

Reply:
Codacy cannot know. It's just a string, you could write anything there.